tir/authorizationClient.go

```diff
@@ -97,11 +104,13 @@ func (ac AuthorizingHttpClient) handleAuthorization(tirAddress string) (bearerTo
 		return bearerToken, err
 	}
 
-	metaData, err := ac.getMetaData(tirAddress)
-	if err != nil {
-		logging.Log().Warnf("Was not able to get the openid metadata. Err: %v", err)
+	metaDataInterface, hit := common.GlobalCache.IssuersCache.Get(tirAddress)
+	if !hit {
+		logging.Log().Errorf("Was not able to get the openid metadata from address %s. Err: %v", tirAddress, err)
 		return bearerToken, err
```

Comment:
Wouldn't both the bearertoken and err be empty here? Don't we need to return an err if we can't get the metadata?